resource.html

```diff
@@ -52,6 +52,7 @@ <h2 id="online-learning--courses">Online Learning &amp; Courses</h2>
   <li><a href="http://chimera.labs.oreilly.com/books/1234000000262/index.html">Programming Javascript Applications</a>: Free Online Book on Javascript</li>
   <li><a href="http://jstherightway.org/">JS: The Right Way</a>: Online reference of Javascript Best Practices</li>
 <li><a href="http://eloquentjavascript.net/">Eloquent Javascript</a>: Free online book on Javascript</li>
+<li><a href="http://lynda.com">Online tutorials covering many subjects in technology and beyond</a></li>
```

Comment:
@Sailor99-GIT Would you mind changing the HTML so the link text is 'Lynda.com' and then the explanation appears afterwards link the other resources? I think you can edit this file by going to this pull request and clicking 'Files changed' and then clicking the edit icon (pencil) in the gray bar